Content.Server/Entry/EntryPoint.cs

```diff
@@ -62,45 +68,66 @@ public override void Init()
 
-            var logManager = IoCManager.Resolve<ILogManager>();
-            logManager.GetSawmill("Storage").Level = LogLevel.Info;
-            logManager.GetSawmill("db.ef").Level = LogLevel.Info;
+                var logManager = IoCManager.Resolve<ILogManager>();
+                logManager.GetSawmill("Storage").Level = LogLevel.Info;
+                logManager.GetSawmill("db.ef").Level = LogLevel.Info;
 
-            IoCManager.Resolve<IConnectionManager>().Initialize();
-            IoCManager.Resolve<IServerDbManager>().Init();
-            IoCManager.Resolve<IServerPreferencesManager>().Init();
-            IoCManager.Resolve<INodeGroupFactory>().Initialize();
-            IoCManager.Resolve<IGamePrototypeLoadManager>().Initialize();
-            _voteManager.Initialize();
+                IoCManager.Resolve<IConnectionManager>().Initialize();
+                IoCManager.Resolve<IServerDbManager>().Init();
+                IoCManager.Resolve<IServerPreferencesManager>().Init();
+                IoCManager.Resolve<INodeGroupFactory>().Initialize();
+                IoCManager.Resolve<IGamePrototypeLoadManager>().Initialize();
+                _voteManager.Initialize();
+            }
         }
 
         public override void PostInit()
         {
             base.PostInit();
 
-            IoCManager.Resolve<ISandboxManager>().Initialize();
-            IoCManager.Resolve<RecipeManager>().Initialize();
-            IoCManager.Resolve<ActionManager>().Initialize();
-            IoCManager.Resolve<BlackboardManager>().Initialize();
-            IoCManager.Resolve<ConsiderationsManager>().Initialize();
-            IoCManager.Resolve<IAdminManager>().Initialize();
-            IoCManager.Resolve<INpcBehaviorManager>().Initialize();
-            IoCManager.Resolve<IAfkManager>().Initialize();
-            IoCManager.Resolve<RulesManager>().Initialize();
-            _euiManager.Initialize();
-
-            IoCManager.Resolve<IGameMapManager>().Initialize();
-            IoCManager.Resolve<IEntitySystemManager>().GetEntitySystem<GameTicker>().PostInitialize();
-            IoCManager.Resolve<IBqlQueryManager>().DoAutoRegistrations();
+            var configManager = IoCManager.Resolve<IConfigurationManager>();
+            var resourceManager = IoCManager.Resolve<IResourceManager>();
+            var dest = configManager.GetCVar(CCVars.DestinationFile);
+            var resPath = new ResourcePath(dest).ToRootedPath();
+            if (dest != "")
+            {
+                var file = resourceManager.UserData.OpenWriteText(resPath.WithName("chem_" + dest));
+                ChemistryJsonGenerator.PublishJson(file);
+                file.Flush();
+                file = resourceManager.UserData.OpenWriteText(resPath.WithName("react_" + dest));
+                ReactionJsonGenerator.PublishJson(file);
+                file.Flush();
+                IoCManager.Resolve<IBaseServer>().Shutdown("Data generation done");
+            }
```

Comment:
this is very ugly but i do not know a better method

Reply:
open to suggestions on a better place to put this. I don't think it should block this PR, but since I plan to add more auto-generated wiki stuff later, we'll get a chance to revisit this.